### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by LukasKalbertodt**
Lots of unrelated improvements. See commit messages.

Fails to deploy because: one commit changes the GH workflows and the deployment scripts such that `fixtures.sql` is not used anymore. But only `ci.yml` is already used by CI. The other scripts are only used once they land in master. So the executed scripts mismatch, leading to an error.

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Helper scripts for development
+
+This directory contains several scripts and docker-compose files that help with development of Tobira.
+None of these files should be used in production.
+
+- `auth-proxy`: an nginx implementing an auth proxy as described in `/docs/auth/all-tobira.md`.
+  Listens on port 3090 and forwards requests to 3080, or for login requests to 3091.
```
Comment (JulianKniephoff) [inline scripts/README.md:7]:
```suggestion
  Forwards requests to port 3090 to port 3080, except for login requests which go to 3091.
```

:shrug:

File: scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- `dev-db`: development PostgreSQL DB. Listens on default Postgres port.
+
+- `login-handler`: a dummy login handler (with fixed login data) as described in `/docs/auth/all-tobira.md`.
+  Listens on port 3091.
+
+- `opencast-cors`: an nginx that's a reverse proxy in front of your local opencast, allowing CORS request.
```
Comment (JulianKniephoff) [inline scripts/README.md:14]:
```suggestion
- `opencast-cors`: an nginx that's a reverse proxy in front of your local Opencast, allowing CORS request.
```

File: scripts/login-handler/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+version: '2'
+
+services:
+  login-handler:
+    container_name: tobira-login-handler-dummy
```
Comment (JulianKniephoff) [inline scripts/login-handler/docker-compose.yml:5]:
```suggestion
```

How married are you to this line? In my experience, `container_name` makes using multiple (different versions) of this service harder. That includes scenarios like switching branches. It's nice for production environments, but I would leave it out for dev scripts like this.

File: scripts/opencast-cors/docker-compose.yml [hunk 4]
```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
   nginx:
-    container_name: opencast-cors-proxy
+    container_name: tobira-opencast-cors-proxy
```
Comment (JulianKniephoff) [inline scripts/opencast-cors/docker-compose.yml:5]: See the other `container_name` comment.

```suggestion
```

File: floof.yaml [hunk 5]
```diff
@@ -5,6 +5,37 @@ default:
...
+    - docker-compose -f opencast-cors/docker-compose.yml up -d
+
+# Stop all those servives again
+stop-scripts:
+  - set-workdir: scripts
+  - concurrently:
+    - docker-compose -f auth-proxy/docker-compose.yml down
+    - docker-compose -f dev-db/docker-compose.yml down
+    - docker-compose -f login-handler/docker-compose.yml down
+    - docker-compose -f opencast-cors/docker-compose.yml down
```
Comment (JulianKniephoff) [inline floof.yaml:32]: Note that `down` is different from `stop`. Down also removes volumes, etc. Is that what we want here?

**Review by JulianKniephoff — COMMENTED** [review]
Some things to discuss, but overall LGTM. :ok_hand:

File: scripts/README.md (hunk 1 above)
Comment (LukasKalbertodt) [reply scripts/README.md:7]: "to port .. to port" sounds a bit meh. But my original also sounds weird. Will figure something out.

**Review by JulianKniephoff — APPROVED** [review]
Alright, everything looks good now! :smile:

**Pull request merged**